tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -15,6 +16,30 @@
 
 
 class ImportCluster(flows.BaseFlow):
+    def _probe_and_mark_provisioner(self, node_list, integration_id):
+        for node in node_list:
+            try:
+                tags = json.loads(
+                    NS._int.client.read(
+                        "nodes/%s/NodeContext/tags" % node
+                    ).value
```

Comment:
Tried this but doesnt work as node context class's render() function doesnt consider the passed node_id and always uses current node's id only 

```
    def render(self):
        self.value = self.value.format(NS.node_context.node_id)
        return super(NodeContext, self).render()
```

If we change that, yes this will work.